Dockerfile.worker

```diff
@@ -16,13 +18,39 @@ RUN python -m textacy download lang_identifier --version 3.0 \
     && python -m spacy download es_core_news_md \
     && python -m spacy download fr_core_news_md
 
-#####
+# Stage 2: Development Stage
 FROM base AS dev
 
+# Install development dependencies
 RUN python -m pip install -r requirements/dev.txt
 
+# Copy only the necessary code to the container
 COPY . .
 
-CMD ["celery", "--app=make_celery.celery_app", "worker", "--loglevel=info"]
+# Set environment variables for development
+ENV FLASK_ENV=development
```

Comment:
Do we need to set this envvar (possibly others?) in the `Dockerfile.api`'s "dev" stage, too? I see that you added it to the prod stage, but not dev.